DRODLib/CharacterCommand.h

```diff
@@ -162,7 +162,8 @@ namespace ScriptFlag
 		PuffImmune = 21,
 		ActivatePlates = 22,
 		PushMonsters = 23,
-		FatalPushImmune = 24
+		FatalPushImmune = 24,
```

Comment:
I realize I didn't understand the semantic of "MonsterTargetIfPlayerIs" before.

The behavior text "Monster Target If Player Is Monster Target" and documentation helps explain this.  Suggest there is a comment added to code, maybe at enumeration #7, explaining what this means.

Alternatively, I have a couple possible naming suggestions, based on how the logic operates.

Option 1. MonsterTargetIfPlayerIsTarget -- reading the original name, at first I was thinking, "If the player is ... what?" I didn't realize "a target" is what was meant.  I thought there might be a parameter for what the player is.  Being explicit in expanding the phrasing helps resolve the ambiguity in reading the name.  To me, it's okay to make these names long if it helps with clarity.

Option 2. MonsterTargetWhenPlayerIsTarget -- I think changing "if" to "when" helps clarify that there is a synchronization aspect to how this behavior works, i.e., the NPC and player being targets are both true or false at the same time.

Option 3. MonsterTargetOncePlayerIsTarget -- if it's impossible for a player to stop being a monster target once in the room they become a target, this name might be clearer.  It helps the architect understand that this behavior gets triggered as soon as the player is flagged as a target, and the behavior will stick for the remainder of the room (unless the behvior setting is changed, of course).

Leaning toward option #3 if I'm understanding correctly how the logic works.

Reply:
I think option 2 is best, since it's possible for the player to regain stealth using the `Set Player Stealth` command. Since it's possible to "go backwards", using when best communicates the syncronization, as you said.